components/group-avatar.tsx

```
@@ -159,8 +159,13 @@ export const GroupAvatar = memo(function GroupAvatar(props: {
         return theme.avatarSize.md
       case "lg":
         return theme.avatarSize.lg
+      case "xl":
+        return theme.avatarSize.xl
+      case "xxl":
+        return theme.avatarSize.xxl
       default:
-        return theme.avatarSize.md
+        const _ensureNever: never = size
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)